tests/py/unit/test_packages_unit.py

```
@@ -0,0 +1,242 @@
+        with patch("trellis.bundler.packages.CACHE_DIR", tmp_path):
+            with patch("trellis.bundler.packages.get_packages_hash", return_value="new123"):
+                with patch("trellis.bundler.packages.ensure_bun") as mock_bun:
+                    mock_bun.return_value = tmp_path / "bun"
+
+                    with patch("subprocess.run") as mock_run:
+                        mock_run.return_value = MagicMock(returncode=0)
+
+                        # Create node_modules that bun install would create
+                        ws = tmp_path / "workspaces" / "new123"
+
+                        def create_node_modules(*args, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `args`

(ARG001)